lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
Does the card require that if two or more expectations exist within a block that they have unique messages, or just that they have messages?